MIEngine.mdd.nuspec

```diff
@@ -14,6 +14,7 @@
   <files>
     <file src="drop\Release\ReferenceAssemblies\*.xml" target="ref\dotnet" />
     <file src="drop\Release\ReferenceAssemblies\Microsoft.DebugEngineHost.dll" target="ref\dotnet" />
+    <file src="drop\Release\ReferenceAssemblies\Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.DesignTime.dll" target="ref\dotnet" />
```

Comment:
Would it be difficult to make a separate nuget package instead? This guy makes sense to be in its own package so that other things can easily reference it.